crates/red_knot_python_semantic/resources/mdtest/stubs/ellipsis.md

```diff
@@ -0,0 +1,42 @@
+# Ellipsis
+
+## Function and Methods
+
+For default values the ellipsis literal `...` can be used.
+
+```py
+from typing import Dict
+
+def f(x: int = ...) -> None: ...
+def f2(x: Dict = ...) -> None: ...
```

Comment:
importing `typing.Dict` seems like an extraneous wrinkle for these tests. since there's no special behavior around it here.
```suggestion
def f(x: int = ...) -> None: ...
def f2(x: dict = ...) -> None: ...
```